demo/api/src/auth/auth.module.ts

```diff
@@ -1,21 +1,26 @@
-import { createAuthResolver, createCometAuthGuard, createStaticAuthedUserStrategy } from "@comet/cms-api";
+import {
+    createAuthResolver,
+    createCometAuthGuard,
+    createStaticAuthedUserStrategy,
+    CURRENT_USER_LOADER,
+    UserPermissionsCurrentUserLoader,
+} from "@comet/cms-api";
 import { Module } from "@nestjs/common";
 import { APP_GUARD } from "@nestjs/core";
 
 import { CurrentUser } from "./current-user";
+import { staticUsers } from "./static-users";
 import { UserService } from "./user.service";
 
 @Module({
     providers: [
+        {
+            provide: CURRENT_USER_LOADER,
+            useClass: UserPermissionsCurrentUserLoader,
+        },
```

Comment:
This is the default case, if there's not custom user loader? Can't the module provide it in the library itself?

Reply:
Good idea and it solves all other issues as well!